File: songshizz_bot/Songwhip/Models/Artist.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Songwhip.Models;

public class Artist
{
    [JsonPropertyName("type")]
    public string Type { get; set; }

    [JsonPropertyName("id")]
    public int Id { get; set; }

    [JsonPropertyName("path")]
    public string Path { get; set; }

    [JsonPropertyName("name")]
    public string Name { get; set; }

    [JsonPropertyName("sourceUrl")]
    public string SourceUrl { get; set; }

    [JsonPropertyName("sourceCountry")]
    public string SourceCountry { get; set; }

    [JsonPropertyName("url")]
    public string Url { get; set; }

    [JsonPropertyName("image")]
    public string Image { get; set; }

    [JsonPropertyName("createdAt")]
    public DateTime CreatedAt { get; set; }

    [JsonPropertyName("updatedAt")]
    public DateTime UpdatedAt { get; set; }

    [JsonPropertyName("refreshedAt")]
    public DateTime RefreshedAt { get; set; }

    [JsonPropertyName("linksCountries")]
    public List<string> LinksCountries { get; set; }

    [JsonPropertyName("links")]
    public Links Links { get; set; }

    [JsonPropertyName("description")]
    public string Description { get; set; }

    [JsonPropertyName("serviceIds")]
    public ServiceIds ServiceIds { get; set; }
}

File: modules/songwhip_api/models/artist.go
package models

import "time"

type Artist struct {
	Type           string     `json:"type"`
	ID             int        `json:"id"`
	Path           string     `json:"path"`
	Name           string     `json:"name"`
	SourceURL      string     `json:"sourceUrl"`
	SourceCountry  string     `json:"sourceCountry"`
	URL            string     `json:"url"`
	Image          string     `json:"image"`
	CreatedAt      time.Time  `json:"createdAt"`
	UpdatedAt      time.Time  `json:"updatedAt"`
	RefreshedAt    time.Time  `json:"refreshedAt"`
	LinksCountries []string   `json:"linksCountries"`
	Links          Links      `json:"links"`
	Description    string     `json:"description"`
	ServiceIds     ServiceIds `json:"serviceIds"`
}
